improver/calibration/dataframe_utilities.py

```diff
@@ -281,7 +281,7 @@ def _prepare_dataframes(
             DataFrame expected to contain the following columns: forecast,
             blend_time, forecast_period, forecast_reference_time, time,
             wmo_id, percentile, diagnostic, latitude, longitude, period,
-            height, cf_name, units. Any other columns are ignored.
+            height, cf_name, units and experiment. Any other columns are ignored.
```

Comment:
It seems like `altitude` is required as well? From: https://github.com/metoppv/improver/blob/a5cb842c30e580f48c5aa6ad35fa8a58584b92fc/improver/calibration/dataframe_utilities.py#L360

Reply:
Yes, you're correct. I've added `altitude` to the list.